data/spells/lib/spells.lua

```diff
@@ -296,3 +296,57 @@ function Creature:addDamageCondition(target, conditionType, listType, damage, ti
 	target:addCondition(condition)
 	return true
 end
+
+function Creature:addPartyCondition(combat, variant, condition, baseMana)
+	local position = self:getPosition()
+	local party = self:getParty()
+	if not party then
+		self:sendTextMessage(MESSAGE_STATUS_SMALL, "No party members in range.")
+		position:sendMagicEffect(CONST_ME_POFF)
+		return false
+	end
+
+	local members = party:getMembers()
+	members[#members + 1] = party:getLeader()
+
+	if members == nil or type(members) ~= 'table' or #members <= 1 then
```

Comment:
It's been a long time since I've used Party feature in Tibia and by that I mean I don't know if it is possible to just have one person at a Party (i.e. a player invites another player (which accepts the request), and then the leader (or the invitee) leaves the party, and the user which didn't leave receives the leadership and now its a party of just 1 member).

Although I tested in TFS, and as you said it is only possible to have 2 or more members in a party, so I'll just change it accordingly.